testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -25,9 +28,59 @@ const STATUS_RESP_TRUE: &str = "success";
 const STATUS_RESP_NOT_COMMITTED: &str = "abort_by_response";
 const STATUS_RESP_POST_CONDITION: &str  = "abort_by_post_condition";
 
+pub const PATH_MEMPOOL_TX_SUBMIT: &str = "new_mempool_tx";
+pub const PATH_BLOCK_PROCESSED: &str = "new_block";
+
 impl EventObserver {
 
-    fn send(&mut self, filtered_events: Vec<&(bool, Txid, &StacksTransactionEvent)>, chain_tip: &ChainTip) {
+    fn send_payload(&self, payload: &serde_json::Value, path: &str) {
+        let endpoint = format!("{}{}{}",
+                               &self.endpoint,
+                               if self.endpoint.ends_with("/") { "" } else { "/" },
```

Comment:
Um...is the sidecar going to crash or misbehave if we don't feed it a path that ends in `/`?  If so, then this is a sidecar bug.

Reply:
No, but if we make a request to `localhost:4000new_block` instead of `localhost:4000/new_block`, we'll have problems.

Reply:
Shouldn't the leading `/` be part of the request path constant then?

Reply:
This check just makes it so that you can supply either `http://localhost:4000/` or `http://localhost:4000` without the setup complaining. This could be moved to the config instead initialization.

Reply:
Addressed in c51a4e4cb